gradle/libs.versions.toml

```diff
@@ -4,15 +4,19 @@ ideProbe = "0.36.1"
 powerMock = "2.0.9"
 
 [libraries]
+aspectj-postCompileWeaving = "io.freefair.gradle:aspectj-plugin:6.4.3.1"
```

Comment:
I'd keep all these Gradle plugins next to each other, visually separated from other "actual" libraries